Solutions/Corvus.Json.Specs/Features/JsonModel/Arrays/JsonArrays.feature

```diff
@@ -6,129 +6,185 @@ Scenario Outline: Remove items from a JsonElement backed JsonArray
 	When I remove the item at index <itemIndex> from the <jsonValueType>
 	Then the <jsonValueType> should equal <result>
 
-	Examples:
-		| jsonValueType | itemIndex | value             | result         |
-		| JsonArray     | 0         | ["foo", "bar", 3] | ["bar", 3]     |
-		| JsonAny       | 0         | ["foo", "bar", 3] | ["bar", 3]     |
-		| JsonNotAny    | 0         | ["foo", "bar", 3] | ["bar", 3]     |
-		| JsonArray     | 1         | ["foo", "bar", 3] | ["foo", 3]     |
-		| JsonAny       | 1         | ["foo", "bar", 3] | ["foo", 3]     |
-		| JsonNotAny    | 1         | ["foo", "bar", 3] | ["foo", 3]     |
-		| JsonArray     | 2         | ["foo", "bar", 3] | ["foo", "bar"] |
-		| JsonAny       | 2         | ["foo", "bar", 3] | ["foo", "bar"] |
-		| JsonNotAny    | 2         | ["foo", "bar", 3] | ["foo", "bar"] |
+Examples:
+	| jsonValueType | itemIndex | value             | result         |
```

Comment:
I guess it was like this when you got here, but this has highlighted that SpecFlow has done its usual "I don't care what your settings say, I'm using tabs, not spaces" thing. If we do want to keep this, we should probably do that thing where we tell GitHub that we interpret them as 4-space tabs.

Reply:
Added a .editorconfig file in the feature folder to treat tabs as 4 char spaces.